com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/util/TimeProfiler.java

```diff
@@ -178,21 +158,15 @@ private static void saveHTML(String jsonReport, File reportFile) throws IOExcept
             throw new IOException("Error while reading time report template: " + e.toString());
         }
         String templateString = writer.toString();
-
-        HashMap<String, Object> ctx = new HashMap<String, Object>();
-        ctx.put("json-data", jsonReport);
-
-        Template template = Mustache.compiler().compile(templateString);
-        StringWriter sw = new StringWriter();
-        template.execute(ctx, sw);
-        sw.flush();
-
-        fileHTMMLWriter.write(sw.toString());
-        fileHTMMLWriter.close();
+        String token = "{{json-data}}";
+        int tokenIndex = templateString.indexOf(token);
+        String beforeToken = templateString.substring(0, tokenIndex - 1);
+        String afterToken = templateString.substring(tokenIndex + token.length()+1);
+        return new String[]{beforeToken, afterToken};
```

Comment:
Instead of using Mustache we manually split template into 2 parts "before" and "after token"